docs/api/covidcast-signals/doctor-visits.md

```diff
@@ -51,6 +51,11 @@ States.
 
 Standard errors are not available for this data source.
 
+Due to changes in medical-seeking behavior on holidays, this data source has
+upward spikes in CLI around major holidays (i.e. Memorial Day, July 4, Labor
```

Comment:
```suggestion
upward spikes in the fraction of doctor's visits that are COVID-related around major holidays (e.g. Memorial Day, July 4, Labor
```

1. i.e. is "that is", i.e. to rephrase something another way, whereas e.g. is for examples, e.g. a list of holidays
2. looks like this doc doesn't define CLI by this point, so may as well be more explicit

Reply:
You should be an English professor!